src/imageLoader/wadors/metaDataManager.js

```diff
@@ -11,7 +71,26 @@ function add(imageId, metadata) {
 function get(imageId) {
   const imageURI = imageIdToURI(imageId);
 
-  return metadataByImageURI[imageURI];
+  let metadata = metadataByImageURI[imageURI];
+
+  if (!metadata) {
+    // in this case try see if the imageId corresponds to multiframe
+    const imageIdFrameless = imageURI.slice(
+      0,
+      imageURI.indexOf('/frames/') + 8
+    );
+    const frame = parseInt(
+      imageURI.slice(imageURI.indexOf('/frames/') + 9),
+      10
+    );
```

Comment:
Does this mean only one digit frames is accepted? like would this fail for `/frames/113` ?

Reply:
No. The slice function with one parameter will get all the text after a given index. imageURI.indexOf('/frames/') + 9 is the first caracter after the slash. In this case t will return '113'